backend/app/server/teams.go

```diff
@@ -250,36 +264,146 @@ func (server *Server) CreateTeam(ctx *gin.Context) {
 func (server *Server) UpdateTeam(ctx *gin.Context) {
 	session := sessions.Default(ctx)
 	userId := session.Get("userId")
+
 	if userId != nil {
 		var team database.DBTeam
-		err := ctx.ShouldBindJSON(&team)
+		err := ctx.ShouldBindJSON(&team) // "message incoming data to this struct"
+		
 		if err != nil {
 			logger.Error("UpdateEvent Request ShouldBindJSON error: %v", err)
 			ctx.Status(http.StatusBadRequest)
 			return
 		}
-		team, err = database.UpdateTeam(team)
+
+		updatedTeam, err := database.UpdateTeam(team)
 		if err != nil {
-			logger.Error("Error calling database.UpdateEvent: %v", err)
+			logger.Error("Error calling database.UpdateTeam: %v", err)
 			ctx.Status(http.StatusInternalServerError)
 		} else {
-			ctx.JSON(http.StatusOK, team)
+			ctx.JSON(http.StatusOK, updatedTeam)
 		}
 	} else {
 		ctx.Status(http.StatusUnauthorized)
 	}
 }
 
+func (server *Server) RemoveTeamMember(ctx *gin.Context) {
+	session := sessions.Default(ctx)
+	userId := session.Get("userId")
+	if userId == nil {
+		ctx.Status(http.StatusUnauthorized)
+		return
+	}
+
+	var MemberPayload MemberPayload
+
+	if err := ctx.ShouldBindJSON(&MemberPayload); err != nil {
+		ctx.JSON(http.StatusBadRequest, gin.H{"error": err.Error()})
+		return
+	}
+
+	uuidTeamId := convert.StringToUUID(MemberPayload.TeamId)
+	uuidMemberId := convert.StringToUUID(MemberPayload.MemberId)
+
+	removedMember, err := database.RemoveTeamMember(uuidTeamId, uuidMemberId)
```

Comment:
For security we'll want to make sure the current user is the team owner and has permission to remove the user.  Otherwise anybody could call this and remove users.